dropwizard-demo/src/main/java/com/dep/models/TweetRespons.java

```diff
@@ -0,0 +1,41 @@
+package com.dep.models;
+
+public class TweetRespons {
+    String message;
+    User user;
+    String createdAt;
+    public TweetRespons(String message, String twitterHandle,String name, String profileImageUrl,String createdAt) {
+        this.message = message;
+        this.user = new User(twitterHandle,name,profileImageUrl);
+        this.createdAt = createdAt;
+    }
+
+    public TweetRespons() {
+
+    }
+
+    public void setUser(User user) {
+        this.user = user;
+    }
+
+    public User getUser() {
+        return user;
+    }
+
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    public String getCreatedAt() {
+        return createdAt;
+    }
+
+    public String setCreatedAt(String createdAt) {
+        this.createdAt = createdAt;
+        return createdAt;
+    }
```

Comment:
not used , remove it
same applies for User class

Reply:
These are required in the TweetResponse class to fetch the data.